src/components/ItaliaTheme/Blocks/HighlightedContent/Sidebar.jsx

```diff
@@ -123,7 +127,18 @@ const Sidebar = ({ block, data, onChangeBlock, openObjectBrowser }) => {
         <CheckboxWidget
           id="show_section"
           title={intl.formatMessage(messages.show_section)}
-          value={data.show_section ? data.show_section : false}
+          value={data.show_section ? data.show_section : true}
```

Comment:
Vedo che cambia anche il default di `show_section`. È voluto?

Reply:
... in realtà così com'è non capisco l'espressione, se show_section è false o undefined value diventa true, se show_section è true, diventa true ... leggo male io?